mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -534,7 +534,7 @@ class LinearRegressionSummary private[regression] (
   @transient private val metrics = new RegressionMetrics(
     predictions
       .select(predictionCol, labelCol)
-      .map { case Row(pred: Double, label: Double) => (pred, label) } )
+      .map { case Row(pred: Double, label: Double) => (pred, label) }, model.getFitIntercept)
```

Comment:
Nit: I'd start the second arg on the next line, indented like the first arg `predictions...` to make it clear it's at the same level.